pkg/features/features.go

```diff
@@ -95,4 +95,9 @@ const (
 	// owner: @scheererj @docktofuture
 	// alpha: v1.22.0
 	ReversedVPN featuregate.Feature = "ReversedVPN"
+
+	// AdminKubeConfigRequest enables the AdminKubeConfigRequest endpoint on shoot resources.
+	// owner: @mvladev
+	// alpha: v1.23.0
+	AdminKubeConfigRequest featuregate.Feature = "AdminKubeConfigRequest"
```

Comment:
```suggestion
	AdminKubeconfigRequest featuregate.Feature = "AdminKubeconfigRequest"
```

This is also how k/k officials names it according to https://kubernetes.io/docs/concepts/configuration/organize-cluster-access-kubeconfig/